### PR TITLE
Adding autoscaling for dynamodb

### PR DESCRIPTION
**Pull request by maartenvanderhoef**
This change adds autoscaling for dynamodb.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -287,6 +371,48 @@ variable "dynamodb3_local_secondary_index_map" {
...
+
+variable "dynamodb3_autoscale_max_write_capacity" {
+  type        = number
+  default     = 20
+  description = "DynamoDB autoscaling max write capacity"
+}
+
+variable "dynamodb3_enable_autoscaler" {
+  type        = string
+  default     = "true"
```
Comment (cytopia) [inline variables.tf]: Isn't that supposed to be `true` (without quotes) with Terraform >= 0.12?

Comment (maartenvanderhoef) [reply variables.tf]: Right!

File: dynamodb.tf [hunk 2]
```diff
@@ -27,8 +27,13 @@ module "dynamodb" {
...
 
-  # We can always extend the defaults and add them
-  enable_autoscaler = false
+  autoscale_write_target       = var.dynamodb_autoscale_write_target
+  autoscale_read_target        = var.dynamodb_autoscale_read_target
+  autoscale_min_read_capacity  = var.dynamodb_autoscale_min_read_capacity
+  autoscale_max_read_capacity  = var.dynamodb_autoscale_max_read_capacity
+  autoscale_min_write_capacity = var.dynamodb_autoscale_min_write_capacity
+  autoscale_max_write_capacity = var.dynamodb_autoscale_max_write_capacity
+  enable_autoscaler            = var.dynamodb_enable_autoscaler ? "true" : "false"
```
Comment (cytopia) [inline dynamodb.tf]: Isn't that also supposed to be a boolean instead of a string?

Comment (maartenvanderhoef) [reply dynamodb.tf]: @cytopia https://github.com/cloudposse/terraform-aws-dynamodb/blob/80c2c516265a379a06258d8bf81892602d151637/variables.tf#L137

Comment (cytopia) [reply dynamodb.tf]: ok

**Pull request merged**